core/templates/dev/head/AppInit.ts

```diff
@@ -20,9 +20,110 @@ import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 import { StaticProvider } from '@angular/core';
 
+/* eslint-disable max-len */
+import { AppConstants } from 'app.constants.ts';
```

Comment:
Why are we importing all the constants from all the pages?

Each page should only be using constants that are relevant to it, or constants which it strictly depends on, right?

Reply:
The constants files need to be registered as providers in the module it belongs and since we have only one module, it needs to register all the constants in this module.

Reply:
Hm, does that mean that every page in Oppia loads all the constants, including those for other pages as well?

Also, is this the long-term plan? If not, we should file an issue that explains what the next steps are for modularizing and separating these.

Reply:
No, this is fairly short term, just until modularity is introduced which I'm working on currently. These can be then placed into the module files of their respective features.

Reply:
Is there actually any advantage to getting this in before modularity is in?

If not, we should just wait until modularity is done, and then get this in, so that we aren't importing things where we don't need to.

Reply:
Yes, that sounds good. We can wait.